Doctrine/Orm/Filter/AbstractFilter.php

```diff
@@ -26,6 +27,8 @@
  */
 abstract class AbstractFilter implements FilterInterface
 {
+    use RequestTrait;
```

Comment:
You do not need a trait at all here, and moreover you're using it only once. IMO traits just hide complexity by hiding dependencies and mixing logic.

Moreover, most of the methods you added to the `AbstractFilter` class are not from its responsibility, it should be another class that is responsible to answer to the questions "isPropertyMapped" or "isPropertyNested".